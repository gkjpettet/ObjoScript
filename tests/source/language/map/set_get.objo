var people = {"garry" : 41, "Aoife" : 10}
System.print(people.count) # expect 2
System.print(people["garry"]) # expect 41

people["Maebh"] = 8
System.print(people.count) # expect 3

System.print(people["Fi"]) # expect nothing

var empty = {}
System.print(empty.count) # expect 0
System.print(empty.isEmpty) # expect true