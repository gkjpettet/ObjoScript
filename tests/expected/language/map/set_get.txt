2
41
3
nothing
0
true